.circleci/config.yml

```diff
@@ -47,6 +77,87 @@ jobs:
       - run:
           name: Run Checks
           command: bin/ci-checks-js.sh
+  android-device-checks:
+    docker:
+    - image: circleci/android:api-27-node8-alpha
+    steps:
+      - checkout
+      - run:
+          name: Checkout Gutenberg
+          command: |
+            git submodule init
+            git submodule update --recursive
+      - yarn-install
+      - run:
+          name: Accept Licenses
+          command: |
+            yes | sdkmanager --licenses > /dev/null
+            sleep 10
+            sdkmanager --update
```

Comment:
Is this step still needed? Looking at the build log it doesn't seem to do anything.

Reply:
It was failing to run the gradle command if I didn't include it, I'm not sure if it's still needed at this point, I'll give it a go though